lib/resources/resourceBase.js

```diff
@@ -1,8 +1,8 @@
 'use strict';
 
-const stream    = require('stream');
-const request   = require('request');
 const BBPromise = require('bluebird');
+const Request   = require('request');
+const Stream    = require('stream');
 
 const ResourceBase = function (endpoint, config) {
   this.uri    = config.options.host + endpoint;
```

Comment:
more string interpolation here and on line 29

Reply:
i am going to change these when refactor the resource files so that they use es6 classes